cli/cmd/cli_state.go

```diff
@@ -235,6 +235,10 @@ func (c *cliState) NewClient() error {
 		api.WithHeader("User-Agent", fmt.Sprintf("Command-Line/%s", Version)),
 	}
 
+	if os.Getenv("LW_TRANSPORT_OVERRIDE") != "" {
```

Comment:
Any implication of using it by default?

Reply:
I don't know, perhaps it would be best left as an optional configuration. We could input the value as the time in seconds.